var/spack/repos/builtin/packages/redset/package.py

```diff
@@ -10,12 +10,14 @@ class Redset(CMakePackage):
     """Create MPI communicators for disparate redundancy sets"""
 
     homepage = "https://github.com/ecp-veloc/redset"
-    url      = "https://github.com/ecp-veloc/redset/archive/v0.0.3.zip"
+    url      = "https://github.com/ecp-veloc/redset/archive/v0.0.5.tar.gz"
     git      = "https://github.com/ecp-veloc/redset.git"
 
     tags = ['ecp']
 
     version('master', branch='master')
+    version('0.0.5', sha256='bb2283e4768276949dc70af0fe92c617356bdb60b7d3b6fe5fe90c8fe630e0b2')
+    version('0.0.4', sha256='cdd867ec45da82c11cbd8c7d8464af2861ffe6b9d6f79cd52d9b8ebff458a70e')
     version('0.0.3', sha256='f110c9b42209d65f84a8478b919b27ebe2d566839cb0cd0c86ccbdb1f51598f4')
```

Comment:
Without any additional edits Spack will assume that `0.0.3` is a `.tar.gz`. If that's true, then the hash should probably change, if it's not, you can add a version-specific URL like:

```
version('0.0.3', sha256='f110c9b42209d65f84a8478b919b27ebe2d566839cb0cd0c86ccbdb1f51598f4', url='https://github.com/ecp-veloc/redset/archive/v0.0.3.zip')
```

Reply:
Ah, yep, good call. Updated the hash.